tests/functional/fx_ios_v1_sign_up.js

```diff
@@ -40,20 +40,17 @@ define([
     },
 
     'sign up, verify same browser': function () {
-      var self = this;
-
       return this.remote
         .then(openPage(PAGE_URL, '#fxa-signup-header'))
         .execute(listenForFxaCommands)
         .then(noSuchElement('#customize-sync'))
         .then(fillOutSignUp(email, PASSWORD))
 
         .findByCssSelector('#fxa-confirm-header')
```

Comment:
Worth updating this to use `FunctionalHelpers.testElementExists` while you're in here?